alica_tests/src/test/Expr/include/AssignPayload.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
```

Comment:
can get rid of assign/unassign payload behaviours & move their functionality to the the onInit & before setting success of the NavigateToPick & Drop behaviours respectively